server/datastore/mysql/migrations/tables/20210709124443_CreateActivitiesTable.go

```diff
@@ -0,0 +1,33 @@
+package tables
+
+import (
+	"database/sql"
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20210709124443, Down_20210709124443)
+}
+
+func Up_20210709124443(tx *sql.Tx) error {
+	sql := `
+		CREATE TABLE IF NOT EXISTS activities (
+			id int(10) unsigned NOT NULL AUTO_INCREMENT,
+			created_at timestamp DEFAULT CURRENT_TIMESTAMP,
+			user_id int unsigned NOT NULL,
+			activity_type varchar(255) NOT NULL,
+			details json DEFAULT NULL,
+			PRIMARY KEY (id),
+			FOREIGN KEY (user_id) REFERENCES users (id) ON DELETE CASCADE ON UPDATE CASCADE
```

Comment:
I wonder if we want `ON DELETE SET NULL`? So that we don't lose activities for a user that was deleted. I suppose we would render them with "Deleted User" or something like that in the UI.

Reply:
I'm removing the foreign key for now.